dev/io.openliberty.jakarta.batch.2.0/bnd.bnd

```diff
@@ -0,0 +1,47 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: Java Batch 2.0 API
+Bundle-SymbolicName: io.openliberty.jakarta.batch-2.0; singleton:=true
+Bundle-Description: jakarta.batch 2.0 API (Reference Implementation)
+
+WS-TraceGroup: wsbatch
+
+Private-Package: \
+    com.ibm.jbatch.jakarta.batch.runtime.internal.*
```

Comment:
Not required, but many teams switched their nls files to io.openliberty for new files.